Library/Formula/llvm.rb

```diff
@@ -116,18 +151,74 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    if build.with?("lldb") && build.without?("clang")
-      raise "Building LLDB needs Clang support library."
+    if build.with? "clang"
+      (buildpath/"tools/clang").install resource("clang")
     end
 
-    if build.with? "clang"
+    if build.with? "extra-tools"
+      if build.with? "clang"
+        (buildpath/"tools/clang/tools/extra").install resource("extra-tools")
+      else
+        odie "clang's extra tools must be built with clang"
```

Comment:
Please use `raise` instead of `odie` here (and everywhere else). Please also be consistent with capitalization: When referring to the compiler in text, it should be `Clang` (like in the option descriptions).